strawberryfields/backends/gaussianbackend/backend.py

```diff
@@ -102,24 +118,36 @@ def measure_homodyne(self, phi, mode, select=None, **kwargs):
         Returns:
             float: measured value
         """
-        eps = kwargs.get('eps', 0.0002)
+        if select is not None and shots != 1:
+            raise ValueError("Post selection and multiple shots is not implemented")
+        eps = kwargs.get("eps", 0.0002)
 
         # phi is the rotation of the measurement operator, hence the minus
         self.circuit.phase_shift(-phi, mode)
 
         if select is None:
-            qs = self.circuit.homodyne(mode, eps)[0]
+            if shots == 1:
+                qs = self.circuit.homodyne(mode, eps)[0,0]
+            else:
+                qs = self.circuit.homodyne(mode, eps, shots = shots)[:,0]
```

Comment:
Bad formatting (file needs to be blacked)
```suggestion
                qs = self.circuit.homodyne(mode, eps, shots=shots)[:, 0]
```